src/main/g8/default.properties

```diff
@@ -16,5 +16,5 @@
 name=format
 extension=dat
 package=com.example
-namespaced=no
-daffodil_version=maven(org.apache.daffodil, daffodil-tdml-processor_2.12)
+namespaced=yes
+daffodil_plugin_version=maven(org.apache.daffodil, sbt-daffodil)
```

Comment:
Note that this now initializes "sbt new" projects with the latest version of the sbt-daffodil plugin available in maven (e.g. 1.0.0 once it's released) instead of the latest version of daffodil TDML processor available in maven. This way the new project will use whatever Daffodil version that plugin defaults to (current 3.7.0).

We could additionally get the latest version of daffodil and add something like `daffodilVersion := $daffodil_version$` to the build.sbt template, which would pin the project to that specific daffodil version. But I kind of like the idea that updating to the latest version of the plugin automatically gets you the latest version of Daffodil. And only if your project really depends on a specific version should you pin it by manually setting `daffodilVersion` in build.sbt.

Thoughts?

Reply:
Only further alternative would be have the template include
```
// the daffodil version is taken from the daffodil-sbt plugin by default. 
// to fix it to a specific version, uncomment
// daffodilVersion := "3.7.0" // for the version you require
```
But there's lots of guidance needed about various things one might need in the build.sbt, and 
we don't want to clutter the template too much. 
Maybe just 
```
// the daffodil sbt plugin provides many options
// which are documented here ... link ....
```

Reply:
When this gets released, updating all existing schema projects to use this will help a lot by getting rid of the unnecessary library dependencies like log4j.

Reply:
I like the idea of adding a link to the daffodil sbt plugin documentation, I'm afraid anything we put in here will eventually get outdated

Reply:
> When this gets released, updating all existing schema projects to use this will help a lot by getting rid of the unnecessary library dependencies like log4j.

I've already started this process, it's definitely making things a lot cleaner and regression testing different daffodil versions is much simpler. Now we can do something like `sbt 'set daffodilVersion := "3.7.0"' test`, so no need to manually update `libraryDependency` versions like we do now.